### DISCUSSION TITLE
Graphical iPXE boot using plymouth splash screen possible?

### DISCUSSION THREAD

**Discussion by shredderique** [ltsp/ltsp#722] [Q&A]
Hi everyone,

 I am maintaining a customized build derived from the official Linux Mint ISO. When I deploy my custom ISO via USB boot I get a splash screen during boot on clients, but not if I use iPXE boot.
Is this a general limitation of booting LTSP over network using plymouth or is anything else to be considered to make it work?

Thanks for any hint!

PS: I would not be too concerned, but I am mounting an encrypted $HOME partition on bootup, and users will be confused if they don't see their usual graphical password query.

**Comment by alkisg** [comment]
I just disabled it by default so that it's easier to troubleshoot netbooting issues.
You can re-enable it by adding this in ltsp.conf:

```shell
[common]
KERNEL_PARAMETERS="quiet splash"
```

Then, run `sudo ltsp ipxe` and reboot the clients so that it takes effect.

**Reply by shredderique** [reply]
Thanks for the hint! 
For some reason I had to add it manually in /srv/tftp/ltsp/ltsp.ipxe to make it work... maybe my template ltsp.ipxe is messed up.

**Comment by X-Dimension-88MPH** [comment]
@shredderique
 
Adding KERNEL_PARAMETERS="quiet splash"
to ltsp.conf is also not working for me on a fresh LTSP Installation with Ubuntu 22.04.

What have you changed in your ltsp.ipxe to get it work?

**Comment by shredderique** [comment]
I figured to put it under the [server] section was the only way to make it work!

```
[server]
# For the boot menu
DEFAULT_IMAGE="ZKRD-Linux"
KERNEL_PARAMETERS="quiet splash username=zkrd bootcfg=pxe netcfg=zkrd"
MENU_TIMEOUT=2000

```
